flytekit/models/task.py

```diff
@@ -843,7 +843,7 @@ def from_flyte_idl(cls, pb2: _core_task.DataLoadingConfig) -> "DataLoadingConfig
 
 
 class Container(_common.FlyteIdlEntity):
-    def __init__(self, image, command, args, resources, env, config, data_loading_config=None):
+    def __init__(self, image, command, args, resources, env, config, architecture=None, data_loading_config=None):
```

Comment:
add type hint here? 
```suggestion
    def __init__(self, image, command, args, resources, env, config, architecture: typing.Optional[int] = None, data_loading_config=None):
```

Reply:
or maybe we should just make the default 0/unknown and always fill it in.  either way as long as it works with the backend.

Reply:
Defaulted to 0, the backend changes check for 0/Unknown and don't do anything in that case, so this should work.